examples/introExample/src/main/java/io/realm/examples/intro/MyApplication.java

```diff
@@ -27,5 +27,9 @@ public void onCreate() {
         super.onCreate();
         // Initialize Realm. Should only be done once when the application starts.
         Realm.init(this);
+
+        // In this example, no default configuration is set, so by default, `RealmConfiguration.Builder().build()` is used.
+        // Please note that while this helps when starting out, you might want to set your own custom default configuration.
+        // Generally, either add a migration to handle schema changes, or specify `deleteIfMigrationNeeded()`.
```

Comment:
I would just remove this sentence. IMO the above is fine and this just goes into too much detail for an intro example